Algorithms/Graphs/MST/prim.cpp

```diff
@@ -0,0 +1,100 @@
+#include <iostream>
+using namespace std;
+
+int findMin( int* keys, bool* included, int v){
+    int mini = 9999;
+    int index;
+    int i;
+
+    for( i=0 ; i<v ; i++ ){
+        if( !included[i] && keys[i] < mini){
+            mini = keys[i];
+            index = i;
+        }
+    }
+    return index;
+}
+
+
+
+int main(){
+    cout<<"Enter number of vertices of the graph"<<endl;
+    int v;
+    cin>>v;
+
+    int graph[v][v];
+    int i,j; //Iterators
```

Comment:
should be i, j;

Reply:
@arnav-mandal1234 is referring to the indentation i.e. there should be a space after the comma.

Reply:
Ohh, okay sure

Reply:
These type of things increase readability.